src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -28,26 +28,32 @@
   private static final String EOL = "([ \t]*\n)?";
 
   /**
-   * These jsdocs delete everything except for the `keep` capture group Some regexes contain an
+   * These jsdocs delete everything except for the `keep` capture group. Some regexes contain an
    * empty capture group for uniform handling.
    */
   private static final Pattern[] JSDOC_REPLACEMENTS = {
+    // The <?keep> capture group is intentionally empty because it's not needed.
     Pattern.compile(
-        BEGIN_JSDOC_LINE + "@(extends|implements|type)[ \t]*(\\{[^@]*\\})[ \t]*(?<keep>)" + EOL,
-        Pattern.DOTALL),
+        BEGIN_JSDOC_LINE + "@(extends|implements|type)[ \t]*(\\{[^@]*\\})[ \t]*(?<keep>)" + EOL),
+    // The <?keep> capture group is intentionally empty because it's not needed.
     Pattern.compile(BEGIN_JSDOC_LINE + "@(constructor|interface|record)[ \t]*(?<keep>)" + EOL),
+    // The <?keep> capture group is intentionally empty because it's not needed.
     Pattern.compile(
         BEGIN_JSDOC_LINE
             + "@(private|protected|public|package|const)[ \t]*(\\{.*\\})?[ \t]*(?<keep>)"
             + EOL),
+    // Removes @enum {number} and @enum {string} without a description.
+    Pattern.compile(BEGIN_JSDOC_LINE + "@enum[ \t]*\\{(string|number)\\}[ \t]*(?<keep>\\*\\/|\n)"),
+    // Removes {number} or {string} type from @enum
```

Comment:
I think for this one we should not keep @enum, because tsickle rejects them.

https://github.com/angular/tsickle/blob/693dd61d59cae0ec48e4aa33734afaaa2ed63767/src/jsdoc.ts#L148

Our rules are a bit ad-hoc, so maybe you can make a case for changing it in tsickle, in any case we should be consistent.

Reply:
Changed to always remove @enum.